packages/katello/katello/katello.spec

```diff
@@ -113,18 +96,18 @@ install -m 644 ./manpages/katello-change-hostname.8.gz %{buildroot}/%{_mandir}/m
 BuildArch:  noarch
 Summary:    Common runtime components of %{name}
 
-Requires:       rubygem-highline
-Requires:       %{name}-debug
+Requires: rubygem-highline
+Requires: %{name}-debug
+Requires: qpid-dispatch-router
+%if 0%{?rhel} == 8
+Requires: container-selinux
```

Comment:
we have a fixed pulpcore-selinux now, should we yank that requirement in develop, test it again, and pick that change too?

Reply:
We should. Has that pulpcore-selinux fix been released to 3.9?

Reply:
it did, yes: https://yum.theforeman.org/pulpcore/3.9/el8/x86_64/pulpcore-selinux-1.2.4-1.el8.x86_64.rpm